gpkit/feasibility.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+from .variables import Variable, VectorVariable
+from .substitution import get_constants
+from .varkey import VarKey
+from .posyarray import PosyArray
+
+
+def feasibility_model(model, flavour="max", varname=None, constants=None):
+    """Returns a new GP for the closest feasible point of the current GP.
```

Comment:
I like extra attributes attached to a GP/Model better than returning them in a big tuple.

I guess the other approach could be to return `(GP/Model, extra_attributes_dict)`, but I'm not married to that.